includes/discovery/sensors/temperature/siteboss.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * LibreNMS Temperature Sensor Discovery module for Asentria SiteBoss
+ *
+ * © 2017 Chris A. Evans <thecityofguanyu@outlook.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+echo 'Asentria SiteBoss ';
+
+$tempoids = snmp_walk($device, 'esIndexPC', '-Osqn', 'SITEBOSS-530-STD-MIB');
```

Comment:
Can this not be converted to yaml?

Reply:
I looked at the YAML sensor docs and certainly considered it, but wasn't as confident in my ability to make it work as I was with the PHP files.